utils/m0reportbug

```diff
@@ -651,7 +653,7 @@ binaries() {
 
     mkdir -p $outdir
     ## Note, that we intentionally skip `m0tr.ko' as it's too big (42M).
-    for f in m0mkfs m0d m0ham m0ut m0beck m0betool ${libs[@]}; do
+    for f in m0mkfs m0d m0ham m0ut m0beck m0betool lt-m0client lt-m0touch ${libs[@]}; do
```

Comment:
[* question *] Is there a reason why the rest of the tools in this line don't have `lt-` prefix and `lt-m0client` and `lt-m0touch` have it?

Reply:
review comment incorporated

Reply:
Have you actually checked that with this change `m0reportug` collects binaries if there is a core file for `m0client` or `m0touch`?

Reply:
Yes.
`[root@ssc-vm-6430 support_bundle]# ls -R m0reportbug-cores
m0reportbug-cores:
binaries  var

m0reportbug-cores/binaries:
libgalois-0.1.so  libmotr.so.2  libmotr-ut.so.0  libmotr-xcode-ff2c.so.0  m0beck  m0betool  m0client  m0d  m0ham  m0mkfs  m0touch  m0ut

m0reportbug-cores/var:
crash  log  motr

m0reportbug-cores/var/crash:
core.0.lt-m0client.107106     core.0.lt-m0client.66570.xz  core.0.lt-m0touch.112824.xz
core.0.lt-m0client.112769.xz  core.0.lt-m0touch.107161.xz  core.0.lt-m0touch.66625.xz

m0reportbug-cores/var/log:
seagate

m0reportbug-cores/var/log/seagate:
motr

m0reportbug-cores/var/log/seagate/motr:

m0reportbug-cores/var/motr:
`

Reply:
For testing i have put an assert

`[root@ssc-vm-6430 temp]# git diff
diff --git a/motr/st/utils/helper.c b/motr/st/utils/helper.c
index 4081b7e..b8c5c8b 100644
--- a/motr/st/utils/helper.c
+++ b/motr/st/utils/helper.c
@@ -256,6 +256,7 @@ static int create_object(struct m0_entity *entity)
        M0_LOG(M0_DEBUG, "post create object, obj->ob_attr.oa_pver :"FID_F,
               FID_P(&obj->ob_attr.oa_pver));

+       M0_ASSERT(false);
        m0_op_fini(ops[0]);
        m0_op_free(ops[0]);

diff --git a/utils/m0reportbug b/utils/m0reportbug
index 49fb59c..21c63d6 100755
--- a/utils/m0reportbug
+++ b/utils/m0reportbug
@@ -653,7 +653,7 @@ binaries() {

     mkdir -p $outdir
     ## Note, that we intentionally skip `m0tr.ko' as it's too big (42M).
-    for f in m0mkfs m0d m0ham m0ut m0beck m0betool lt-m0client lt-m0touch ${libs[@]}; do
+    for f in m0mkfs m0d m0ham m0ut m0beck m0betool m0client m0touch ${libs[@]}; do
         s=$(path $f binary)
         if [ -n "$s" ]; then
             ## Strip `.0.0' suffix when copying.
`
We can see m0client and m0touch
`[root@ssc-vm-6430 support_bundle]# ls -R m0reportbug-cores
m0reportbug-cores:
binaries  var

m0reportbug-cores/binaries:
libgalois-0.1.so  libmotr.so.2  libmotr-ut.so.0  libmotr-xcode-ff2c.so.0  m0beck  m0betool  m0client  m0d  m0ham  m0mkfs  m0touch  m0ut

m0reportbug-cores/var:
crash  log  motr

m0reportbug-cores/var/crash:
core.0.lt-m0client.107106     core.0.lt-m0client.66570.xz  core.0.lt-m0touch.112824.xz
core.0.lt-m0client.112769.xz  core.0.lt-m0touch.107161.xz  core.0.lt-m0touch.66625.xz

m0reportbug-cores/var/log:
seagate

m0reportbug-cores/var/log/seagate:
motr

m0reportbug-cores/var/log/seagate/motr:

m0reportbug-cores/var/motr:
`